babel.js

```diff
@@ -37,33 +19,32 @@ module.exports = function (babel) {
     var condition = _.find(attributes, function (attr) {
       return attr.name.name === 'condition';
     });
-
     if (!condition) {
       throwError(errors.IF_WITH_NO_CONDITION, node, file);
     }
 
-    var children = removeLiterals(node.children);
-
-    var notElseTag = function (child) {
-      return child.type !== 'JSXElement' || child.openingElement.name.name !== 'Else';
-    };
-
-    var ifBlock, elseBlock;
+    // normalize JSXText and JSXExpressionContainer to expressions
+    var children = t.react.buildChildren(node);
```

Comment:
So this basically does what removeLiterals did but built-in to babel? Nice find.